template-sample-app/template/src/components/layers/KpiLayer.js

```diff
@@ -1,8 +1,8 @@
 import { useSelector } from 'react-redux';
 import { CartoSQLLayer } from '@deck.gl/carto';
-import { scaleThreshold } from 'd3-scale';
-import { buildQueryFilters } from '@carto/react/api';
 import { selectSourceById } from '@carto/react/redux';
+import { useCartoProps } from '@carto/react/api';
+import { scaleThreshold } from 'd3-scale';
```

Comment:
I have seen you added the d3-scale dependency to 'template-sample-app'. It's quite annoying, but right now we have to duplicate the work, and also add it to skeleton template

Reply:
`d3-scale` was added a time ago, I prefer to use CARTO style functions (`colorBins`).

Will replicate it in skeleton template :+1: